lib/ansible/modules/source_control/git.py

```diff
@@ -977,6 +999,17 @@ def main():
     track_submodules = module.params['track_submodules']
 
     result.update(before=None)
+
+    if archive_fmt not in ['zip', 'tar']:
+        archive_fmt = 'tar'
+
+    if archive:
+        cmd = "%s archive --format=%s --remote=%s --output=%s master" \
```

Comment:
Will this command work on all platforms? I think it is better to use standard python modules for creating archive.